tests/e2e/tests/llm-revert-check_jboss-eap-quickstarts.test.ts

```diff
@@ -0,0 +1,187 @@
+import { expect, test } from '../fixtures/test-repo-fixture';
+import { VSCode } from '../pages/vscode.pages';
+import { SCREENSHOTS_FOLDER, TEST_OUTPUT_FOLDER } from '../utilities/consts';
+import { getOSInfo, getRepoName, generateRandomString } from '../utilities/utils';
+import { providerConfigs } from '../fixtures/provider-configs.fixture';
+import path from 'path';
+import { runEvaluation } from '../../kai-evaluator/core';
+import { prepareEvaluationData, saveOriginalAnalysisFile } from '../utilities/evaluation.utils';
+import { KAIViews } from '../enums/views.enum';
+import { getFileImports } from '../utilities/file.utils';
+
+providerConfigs.forEach((config) => {
+  test.describe(`LLM Revertion tests | ${config.model}`, () => {
+    let vscodeApp: VSCode;
+    let allOk = true;
+    const randomString = generateRandomString();
+    let profileName = `llm-revetion-${randomString}`;
+
+    const kitchenRepoPath = 'jboss-eap-quickstarts/kitchensink';
+    const relativeMemberFileUri =
+      '../../jboss-eap-quickstarts/kitchensink/src/main/java/org/jboss/as/quickstarts/kitchensink/model/Member.java';
+    const relativeKoveyorPath = '../../jboss-eap-quickstarts/kitchensink/.vscode/konveyor';
+
+    const memberFileUri = path.resolve(__dirname, relativeMemberFileUri);
+    const konveyorRepoPath = path.resolve(__dirname, relativeKoveyorPath);
+
+    let beforeTestMemberFileImports: string[];
+    let afterFirstFixMemberFileImports: string[];
+    let afterSecondFixMemberFileImports: string[];
+
+    test.beforeAll(async ({ testRepoData }, testInfo) => {
+      test.setTimeout(3600000);
+      const repoName = getRepoName(testInfo);
+      const repoInfo = testRepoData[repoName];
+
+      vscodeApp = await VSCode.open(repoInfo.repoUrl, repoInfo.repoName, repoInfo.branch);
+      await vscodeApp.closeVSCode();
+      //analyizing only kitchensink is less time consuming then the entire jboss repo
```

Comment:
Can be rephrased to : 

// Only analyzing kitchensink to save time vs. full jboss repo